examples/imdb_cnn_lstm.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import absolute_import
+from __future__ import print_function
+import numpy as np
+np.random.seed(1337)  # for reproducibility
+
+from keras.preprocessing import sequence
+from keras.optimizers import SGD, RMSprop, Adagrad
+from keras.utils import np_utils
+from keras.models import Sequential
+from keras.layers.core import Dense, Dropout, Activation
+from keras.layers.embeddings import Embedding
+from keras.layers.recurrent import LSTM, GRU, SimpleRNN
+from keras.layers.convolutional import Convolution1D, MaxPooling1D
+from keras.datasets import imdb
+
+'''
+    Train a combination of Convoloution1D and LSTM on the IMDB sentiment classification task.
```

Comment:
"Convolution". By the way "combination" is somewhat unclear since it first made me think of an ensemble. A "recurrent convolutional network" might be clearer.